src/Form/Form.js

```diff
@@ -327,11 +359,13 @@ class Form {
     }
     // This removes all listeners to avoid memory leaks and weird behaviours
     destroy () {
+        this.mutObs.disconnect()
         this.removeAllDecorations()
         this.removeTooltip()
         this.forgetAllInputs()
         this.matching.clear()
         this.intObs = null
+        this.device.scanner.forms.delete(this.form)
```

Comment:
This is also new. When `destroy`ing the form we also ensure not to keep track of it in the scanner ledger. I don't particularly like reaching out to the `scanner`, but it's effective.